go/libkb/interfaces.go

```diff
@@ -701,9 +701,9 @@ type TeamAuditor interface {
 type TeamBoxAuditor interface {
 	AssertUnjailedOrReaudit(m MetaContext, id keybase1.TeamID) (didReaudit bool, err error)
 	IsInJail(m MetaContext, id keybase1.TeamID) (bool, error)
-	RetryNextBoxAudit(m MetaContext) (err error)
-	BoxAuditRandomTeam(m MetaContext) (err error)
-	BoxAuditTeam(m MetaContext, id keybase1.TeamID) (err error)
+	RetryNextBoxAudit(m MetaContext) (attempt *keybase1.BoxAuditAttempt, err error)
+	BoxAuditRandomTeam(m MetaContext) (attempt *keybase1.BoxAuditAttempt, err error)
+	BoxAuditTeam(m MetaContext, id keybase1.TeamID) (attempt *keybase1.BoxAuditAttempt, err error)
```

Comment:
do we need to return pointers here?

also these will not be shared by the same object that's in the box audit logs, I noticed that BoxAuditAttempt has some pointers in there, so there might still be some unwanted data sharing. Should we ensure we use deep copy to store these?

Reply:
not really, it was to distinguish the case where the attempt was never even attempted. but I can make a field inside attempt for that

Reply:
ah then my point is moot